.github/workflows/ci.yml

```
@@ -19,3 +31,7 @@ jobs:
           mix credo --strict
       - name: Run Tests
         run: mix test
+      - name: Run Hex Publish test
+        run: mix hex.publish --dry-run
+        env:
+          HEX_API_KEY: ${{ secrets.HEX_API_KEY }}
```

Since we’re just doing a dry run, there’s no need to use the actual key:

```suggestion
          HEX_API_KEY: DRYRUN
```

---

Indeed. So we could just delete this line to keep only
```elixir
- name: Run Hex Publish test
   run: mix hex.publish --dry-run
```

---

is it ok to have this non fatal error ?
```
puublishing package to public repository hexpm.
invalid API key
```